.goreleaser.yaml

```diff
@@ -0,0 +1,47 @@
+project_name: spacelift-vcs-agent
```

Comment:
Since you're not switching the build to Goreleaser in this PR, can you remove this file from the PR? It means when we do the PR to switch to Goreleaser it will be self-contained and won't rely on a change from this PR.